element-hq/element-x-android
# Use new functions exposed by Element Call about PiP

## Content


- Communicate with Element Call about PiP status.
- Also only use eventSink to communicate with the Presenter, instead of having public methods.
- Change WeakReference to an Activity to a listener and update tests.

## Motivation and context


Closes #3326

## Screenshots / GIFs



## Tests



- Change Element Call url to https://pr2573--element-call.netlify.app/
- Start a call and play with the Pip mode
- Note that if the call is in error, the pip mode will not start, but the call will be terminated. This can be tested when entering pip mode just after starting the call (and this is maybe a thing to fix on Element Call side. The call is not in error, but not fully started yet CC @robintown )

Note: using regular (so not deployed) Element Call should not be impacted by this change.

## Tested devices

- OS version(s):

## Checklist
